Week1/js-exercises/ex4-whatsTheTime.js

```diff
@@ -10,9 +10,21 @@
   4. Have the function execute when it 's loading in the browser
 
  */
+// create h1
+let clock = document.createElement('h1');
+clock.style.fontFamily = 'Arial, sans-serif';
+clock.style.textAlign = 'center';
 
+// insert h1 in the document
+document.body.appendChild(clock);
+
+// the function of setting time
 function displayCurrentTime() {
   // your code goes in here
+  let timeNew = new Date();
+  let seet = timeNew.toLocaleTimeString('en-uk');
```

Comment:
That is not the correct format
```suggestion
  let seet = timeNew.toLocaleTimeString('nl-nl');
```

Reply:
sorry for this but as I told you we have few knowledge at that time.
your comments are important to improve my knowledge.